packages/Uploader/src/components/DropZone/DropZone.styles.js

```diff
@@ -26,10 +50,23 @@ export const DropZone = styled.div(
       min-height: auto;
       padding: 0;
     }
+    span[data-pka-anchor="uploader-dropZone-link"] {
+      margin-bottom: 3px;
+    }
+  `,
+  `text-align: center;
```

Comment:
This looks out of place here... is it even working?  If it's for the `DropZone` element it can just go up above on like 15 or something.